websites/A/Animevietsub/presence.ts

```diff
@@ -0,0 +1,99 @@
+const presence = new Presence({
+		clientId: "1016991973531451502",
+	}),
+	strings = presence.getStrings({
```

Comment:
```suggestion
	strings = await presence.getStrings({
```

Reply:
By bad, but you can't use await when it's not in a function. I suggest you to move the func in the UpdateData